app/(unauthenticated)/sign-up/email-sent/page.tsx

```diff
@@ -1,11 +1,15 @@
 "use client";
+import React from "react";
```

Comment:
Is this one needeed? if yes, can we change it to:
`import React, { Suspense } from "react";`